internal/cache/metrics.go

```diff
@@ -47,7 +47,7 @@ func NewCacheRecorder() *CacheRecorder {
 	return &CacheRecorder{
 		cacheEventsCounter: prometheus.NewCounterVec(
 			prometheus.CounterOpts{
-				Name: "gotk_cache_events_total",
+				Name: "gotk_sc_cache_events_total",
```

Comment:
Changing the metric name is a breaking change, what is the reason for this change?

Reply:
there is a conflict in naming with the same metric in `flux/pkg/cache`. I'll remove the SC internal `cache` package to fix this.